### PR TITLE
User documentation for Zero Configuration

### PR DESCRIPTION
**Pull request by ibennetch**
Signed-off-by: Isaac Bennetch bennetch@gmail.com

### CODE REVIEW

**Comment by ibennetch** [comment]
This targets QA_4_3.
I'm looking for additional options about whether this is accurate or if you have better suggestions. Thanks

File: doc/setup.rst [hunk 1]
```diff
@@ -262,6 +262,33 @@ in your own database, or in a central database for a multi-user installation
...
+  phpMyAdmin continues to use the tables from the first database; the user is
+  not prompted to create more tables in the new database.
+
+
+Manual configuration
+--------------------
+
 Please look at your ``./examples/`` directory, where you should find a
 file called *create\_tables.sql*. (If you are using a Windows server,
 pay special attention to :ref:`faq1_23`).
```
Comment (cwittmer) [inline doc/setup.rst]: Please look at your ``./sql/`` directory, where you should find a
file called *create\_tables.sql*. (If you are using a Windows server,
pay special attention to :ref:`faq1_23`).

Comment (lem9) [inline doc/setup.rst]: The directory name changed only in the master branch for the future 4.4.

**Comment by lem9** [comment]
Partially merged, thanks.

**Pull request closed without merging**